tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -3391,6 +3391,9 @@ public function clearSpaceId(): void {
 		if (\count(WebDavHelper::$spacesIdRef) > 0) {
 			WebDavHelper::$spacesIdRef = [];
 		}
+		if (isset(WebDavHelper::$SPACE_ID_FROM_OCIS)) {
+			WebDavHelper::$SPACE_ID_FROM_OCIS = '';
+		}
```

Comment:
review this @saw-jan dai

Reply:
Don't we clear this after each request after we generate the fullUrl? @KarunAtreya

Reply:
I think we clear this after each request. any case where this is to be clean in afterScenario? But yeah i think that we do not need this variable `$SPACE_ID_FROM_OCIS` variable anymore. This was added because we used the same testcode for `core`. We can make an issue to refactor this also. @saw-jan @SwikritiT

Reply:
> I think we clear this after each request. any case where this is to be clean in afterScenario? But yeah i think that we do not need this variable `$SPACE_ID_FROM_OCIS` variable anymore. This was added because we used the same testcode for `core`. We can make an issue to refactor this also. @saw-jan @SwikritiT

I actually removed this before but this scenario was failing: apiSpacesShares/shareUploadTUS.feature:248
later found out that $SPACE_ID_FROM_OCIS was not reset in the scenario apiSpacesShares/shareUploadTUS.feature:248 so the request was send to the space id from previous scenario resulting 409 status error

Reply:
> We can make an issue to refactor this also.

yeah, should be on the list